website/src/arcade/share.ts

```diff
@@ -0,0 +1,10 @@
+export const copyToClipboard = (value: string, cb?: () => void) =>
+  navigator.clipboard
+    .writeText(value)
+    .then(cb)
+    .catch(() => null);
```

Comment:
if for some reason we can't copy this, worth throwing up a toast or something? fine as a follow on or as a todo later